src/index.js

```diff
@@ -26,12 +26,20 @@ function syncEvent(node, eventName, newEventHandler) {
   }
 }
 
-export default function (CustomElement, opts) {
+export default function (CustomElementOrTagName, opts) {
   opts = assign({}, defaults, opts);
-  if (typeof CustomElement !== 'function') {
-    throw new Error('Given element is not a valid constructor');
+  if (typeof CustomElementOrTagName !== 'function' && typeof CustomElementOrTagName !== 'string') {
+    throw new Error('Given element is not a valid constructor or tag name');
   }
-  const tagName = (new CustomElement()).tagName;
+
+  let CustomElementConstructor = CustomElementOrTagName
+  if (typeof CustomElementOrTagName === 'string') {
+    CustomElementConstructor = document.createElement(CustomElementOrTagName).constructor
+  }
+
+  const tagName = typeof CustomElementOrTagName === 'string'
+      ? CustomElementOrTagName
```

Comment:
You wouldn't need this part if looking up via `customElemenst.get()` but you would need to use the `is` attribute if it exists and fallback to `tagName` if not.